doc/releases/changelog-0.37.0.md

```diff
@@ -20,43 +20,88 @@
 
 <h4>Add noise models to your quantum circuits 📺</h4>
 
-* A new `qml.noise` module which contains utility function for building `NoiseModels` 
-  and an `add_noise` tranform for addding it to quantum circuits.
+* A new `qml.noise` module has been added, which contains utilities for building `NoiseModel`s 
+  and an `add_noise` transform for adding it to quantum circuits.
   [(#5674)](https://github.com/PennyLaneAI/pennylane/pull/5674)
   [(#5684)](https://github.com/PennyLaneAI/pennylane/pull/5684)
   [(#5718)](https://github.com/PennyLaneAI/pennylane/pull/5718)
 
+  Under the hood, PennyLane's approach to noise models is insertion-based, meaning that noise is simulated
+  by *inserting* additional operators (gates) that describe the noise into the quantum circuit itself. 
+  Creating a `NoiseModel` boils down to defining Boolean conditions under which specific noisy operations 
+  are inserted. There are several ways to specify conditions for adding noisy operations: 
+
+  * `qml.noise.op_eq(op)`: if the operator `op` is encountered in the circuit, add noise.
+  * `qml.noise.op_in(ops)`: if any operators in `ops` are encountered in the circuit, add noise.
+  * `qml.noise.wires_eq(wires)`: if an operator is applied to `wires`, add noise.
+  * `qml.noise.wires_in(wires)`: if an operator is applied to any wire in `wires`, add noise.
+  * custom noise conditions: custom conditions can be defined as functions decorated with `qml.BooleanFn` that return a Boolean value. For example, the following function will insert noise if a `qml.RY` operator is encountered with an angle of rotation that is less than `0.5`:
+
+    ```python
+    @qml.BooleanFn
+    def c0(op):
+        return isinstance(op, qml.RY) and op.parameters[0] < 0.5
+    ```
+
+  `op_eq`, `op_in`, `wires_eq`, and `wires_in` can also be combined together with `&`, `and`, `|`, etc.
+  Once the conditions under which noise is to be inserted have been stated, we can specify exactly what 
+  noise is inserted with the following:
+
+  * `qml.noise.partial_wires(op)`: insert `op` on the wires that are specified by the condition that triggers adding this noise
+  * custom noise operations: custom noise can be specified by defining a standard quantum function like below.
+  
+  ```python
+  def n0(op, **kwargs):
+      qml.RY(op.parameters[0] * 0.05, wires=op.wires)
+  ```
+
+  With that, we can create a `qml.NoiseModel` object whose argument must be a dictionary mapping conditions
+  to noise:
+
+  ```python
+  c1 = qml.noise.op_eq(qml.X) | qml.noise.wires_in([0, 1])
```

Comment:
```suggestion
  c1 = qml.noise.op_eq(qml.X) & qml.noise.wires_in([0, 1])
```
Needed this else the example fails because there is an `AmplitudeDamping` that has more than one wire passed.